### PR TITLE
Reworking Data Analyst and Data Vis for August 2023 recruitment

### PR DESCRIPTION
**Pull request by Gordonei**
Main objectives:
* refresh data analyst position:
  * @demainou expanded upon the questions asked
  * Changed from focusing on Water and Sanitation to Urban Mobility, to keep things interesting
* simplify and reduce data vis workflow, while still testing the variety of skills

### CODE REVIEW

**Review by demainou — APPROVED** [review]
happy

File: README.md [hunk 1]
```diff
@@ -145,15 +151,14 @@ Using the [`sr_hex.csv.gz`](https://cct-ds-code-challenge-input-data.s3.af-south
...
-6.    Design Principles: In a separate markdown document, titled `visualisation-design-choices.md`, please provide a brief, point form explanation for why you have chosen certain colours (e.g. for legends), fonts, the layout or anything else that will help us understand your thinking in designing the data visualisation / dashboard to answer the question.
-7.    Publish your work using an online service such as https://pages.github.com/ or any other means you are familiar with.  Anyone with an Internet connection and a modern browser such as Google Chrome, Mozilla Firefox or Microsoft Edge, should be able to see the end product and interact with it. Please reference the published link to your visualisation tool in the `README.md` of your repository.
+1. A chart (plot) or charts (plots) that helps to answer the above question.
+2. A minimalist cartographic map with identifiable landmark features (e.g. major roads, railways, etc.) and some representation of the data.
+3. Make (1) and (2) interactive in some manner, so as to allow users to explore the data and uncover insights.
```
Comment (tockards) [inline README.md]:
```suggestion
3. Make (1) and (2) interactive in some manner, so as to allow users to explore the data and uncover insights. The following example [Map with "range" sliders](https://observablehq.com/d/a040753103477386) demostrates an interactive map. However, you're not limited to this example – feel free to explore other interactive approaches.
```

Comment (tockards) [reply README.md]: Added an example around the basic interactivity we expect.

**Review by tockards — COMMENTED** [review]
Added a basic example for map interactivity.

**Pull request merged**